src/module/entities/validator/v1.js

```diff
@@ -14,8 +14,12 @@ module.exports = (req) => {
 			req.checkBody('name').exists().withMessage('required name ')
 		},
 		update: function () {
-			req.checkBody('externalId').exists().withMessage('required externalId')
 			req.checkParams('_id').exists().withMessage('required _id')
+			if (req.body.metaInformation && !req.body.metaInformation.externalId) {
```

Comment:
@MallanagoudaB Based 0n our discussion please change the condition , test it well pls